configure/configuring-quotas.adoc

```diff
@@ -0,0 +1,46 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="configuring-quotas"]
+= Configuring quotas
+:context: configuring-quotas
+
+toc::[]
+
+As an administrator, you can use {product-title} to configure quotas and limit access to GPUs to optimize resource allocation and system throughput for user workloads.
```

Comment:
GPU is an example here but Kueue could be used to limit cpu/memory/ephermeral-storage. 

Can we make this more clear?